asm-iap/Kptfile

```diff
@@ -50,6 +50,20 @@ openAPI:
         setter:
           name: anthos.servicemesh.trustDomainAliases
           listValues:
+    io.k8s.cli.setters.gcloud.container.cluster.ingress.httpPort:
+      type: integer
+      x-k8s-cli:
+        setter:
+          name: gcloud.container.cluster.ingress.httpPort
+          value: 31224
```

Comment:
Try putting 31224 and 31223 in quotes.

Reply:
thx, that did the trick.  apparently ints are bad.

Reply:
I think that very unintuitively, the values that kpt uses are all strings regardless of what type you set. Using integer is referring to the value of the yaml, not in the text file.